package.json

```diff
@@ -38,11 +38,18 @@
     "core-js": "^3.6.5",
     "filesize": "^6.1.0",
```

Comment:
filesize dep can be removed, right?

Reply:
as you've switched to our `oc-resource-size` component 😍

Reply:
It's an ODS peer dep https://github.com/owncloud/owncloud-design-system/blob/f0121d7cf5341843f01852885d1cc103f21bfac1/package.json#L145

Reply:
Ah, true. Sorry for the noise 😁